synapse/server.py

```diff
@@ -233,15 +234,8 @@ def __init__(
         self._instance_id = random_string(5)
         self._instance_name = config.worker_name or "master"
 
-        self.clock = Clock(reactor)
         self.distributor = Distributor()
```

Comment:
Should the same change be made to `distributor`?

Alternately we should potentially get rid of the `get_distributor` method. 🤷

Reply:
Should I make that change?

Reply:
I think so! 👍